protected/controllers/survey/Create.php

```diff
@@ -0,0 +1,27 @@
+<?php
+declare(strict_types=1);
+
+namespace prime\controllers\survey;
+
+use prime\interfaces\AccessCheckInterface;
+use prime\models\ar\Permission;
+use prime\models\ar\Survey;
+use prime\models\survey\SurveyForCreate;
+use yii\base\Action;
+
+class Create extends Action
+{
+    public function run(
+        AccessCheckInterface $accessCheck,
+    ) {
+        $model = new SurveyForCreate();
+        $accessCheck->checkPermission(new Survey(), Permission::PERMISSION_CREATE);
```

Comment:
Should we overload this type of checking to support the real model? Instead of creating a different model type for this check?

I've done this for other models via the repository and an interface; but for this case it seems more like a generic alias scenario. (Since the model has no identity)